packages/flutter/lib/src/material/mergeable_material.dart

```diff
@@ -537,16 +543,9 @@ class _MergeableMaterialState extends State<MergeableMaterial> with TickerProvid
       if (_children[i] is MaterialGap) {
         assert(slices.isNotEmpty);
         widgets.add(
-          Container(
-            decoration: BoxDecoration(
-              color: Theme.of(context).cardColor,
-              borderRadius: _borderRadius(i - 1, widgets.isEmpty, false),
-              shape: BoxShape.rectangle,
-            ),
-            child: ListBody(
-              mainAxis: widget.mainAxis,
-              children: slices,
-            ),
+          ListBody(
+            mainAxis: widget.mainAxis,
+            children: slices,
           ),
         );
```

Comment:
I thought to refactor the code by removing this but as I said I was facing one issue. I have pushed the changes, so I can explain the issue.

Reply:
This was the result if I was trying to refactor more, the SizedBox was getting a different color. I was not able to find out the reason, I'll try to look into it if this implementation needs more refactoring.

<img src="https://user-images.githubusercontent.com/34301187/92299811-67cd3280-ef73-11ea-9a63-0c022fe7e379.png" width=200>